xml/System.Text.Json/Utf8JsonReader.xml

```diff
@@ -454,10 +617,13 @@
         <Parameter Name="value" Type="System.Decimal" RefType="out" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="value">When this method returns, contains the parsed value.</param>
```

Comment:
This is a bit awkward to read.

Reply:
This is what our guidelines say for **out** parameters. See others here:
https://github.com/dotnet/dotnet-api-docs/search?q=%22when+this+method+returns%22&unscoped_q=%22when+this+method+returns%22

We could come with a new convention if it's awkward.